internal/pkg/pct/install.go

```diff
@@ -92,6 +96,53 @@ func (p *PctInstaller) Install(templatePkg string, targetDir string, force bool)
 	return namespacedPath, nil
 }
 
+func (p *PctInstaller) InstallClone(gitUri string, targetDir string, tempDir string, force bool) (string, error) {
```

Comment:
Similar to my previous comment:

```suggestion
func (p *PctInstaller) InstallClone(gitUri, targetDir, tempDir string, force bool) (string, error) {
```